lib/service/storage/elasticsearch.js

```diff
@@ -2965,7 +2965,7 @@ class ElasticSearch extends Service {
     });
 
     if (body.length < 1) {
-      throw kerror.get("unknown_index_collection");
+      throw kerror.get(`unknown_index_collection : ${alias}`);
```

Comment:
This method does not work like a console log, you must edit the associated error code description message to add your variable (https://github.com/kuzzleio/kuzzle/blob/75cb185aed40c3d4cf6dad33eac905404c723549/lib/kerror/codes/1-services.json#L88). You could find such a message syntax here https://github.com/kuzzleio/kuzzle/blob/75cb185aed40c3d4cf6dad33eac905404c723549/lib/kerror/codes/1-services.json#L100